modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -25,22 +25,33 @@ function perflab_aea_audit_enqueued_scripts() {
 		global $wp_scripts;
 		$enqueued_scripts = array();
 
-		foreach ( $wp_scripts->queue as $handle ) {
-			$src = $wp_scripts->registered[ $handle ]->src;
-			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+		foreach ( $wp_scripts->done as $handle ) {
+			$script = $wp_scripts->registered[ $handle ];
+
+			if ( $script->src && ! strpos( $script->src, 'wp-includes' ) ) {
+
+				// Add any extra data (inlined) that was passed with the script.
+				$inline_size = 0;
+				if ( ! empty( $script->extra ) && ! empty( $script->extra['after'] ) ) {
+					foreach ( $script->extra['after'] as $extra ) {
+						$inline_size += ( is_string( $extra ) ) ? mb_strlen( $extra, '8bit' ) : 0;
+					}
+				}
+
+				$path               = perflab_aea_get_path_from_resource_url( $script->src );
 				$enqueued_scripts[] = array(
-					'src'  => $src,
-					'size' => perflab_aea_get_resource_file_size( perflab_aea_get_path_from_resource_url( $src ) ),
+					'src'  => $script->src,
+					'size' => perflab_aea_get_resource_file_size( $path ) + $inline_size,
 				);
 			}
 		}
 		set_transient( 'aea_enqueued_front_page_scripts', $enqueued_scripts, 12 * HOUR_IN_SECONDS );
 	}
 }
-add_action( 'wp_print_scripts', 'perflab_aea_audit_enqueued_scripts' );
+add_action( 'wp_footer', 'perflab_aea_audit_enqueued_scripts', PHP_INT_MAX );
 
 /**
- * Audit enqueued styles in the frontend. Ignore /wp-includes styles.
+ * Audit enqueued and printed styles in the frontend. Ignore /wp-includes styles.
```

Comment:
Why would we ignore `wp-includes`? Gutenberg for instance adds a couple of stylesheets that arent exactly optimized. My opinion is that we should give the user the full picture, even if it means flagging WP for enqueuing unnecessary stylesheets.

Reply:
@dainemawer those assets are included by default by wp, and this module focus on an overall overview of external assets being enqueued (plugins, theme), I think it would make more noise to include wp-includes, since most of the consumers won't be techies and won't know how to react on it.
I agree that going forward and as we iterate over this module, we can show more useful information, at that point, we can add more info (like wp-includes) and tips on how to improve it, but for now, I would just ignore them.

Reply:
Thats a fair statement, thanks @manuelRod !